students/Brayden Borzyskowski/Lab2/Lab2Part3/Lab2Part3.ino

```diff
@@ -0,0 +1,39 @@
+const int apin = 0;
+const float aref = 5.0; //test voltage output ie.. 5.13
+const int adc_max = 1023;
+const float temp_slope = 100;
+int val = 0;
+const int LED = 9;
+
+
+
+void setup() {
+  // put your setup code here, to run once:
+  Serial.begin(9600);
+  //analogReference(EXTERNAL); - then set const float aref to 3.37 or whatever aref value 
+  //is outputting when using the micrometer to measure the output voltage
+ 
+}
+
+void loop() {
+  // put your main code here, to run repeatedly:
+
+  // Read Sensor
+  val = analogRead(apin);
+
+  // Convert to voltage
+  float voltage = val * aref / adc_max;
+  //float voltage = map(val, 0, adc_max, 0, aref);
+  //Serial.println(voltage);
+
+  // Convert to temperature
+  float temp = voltage * temp_slope;
+
+  //Print to serial
+  //Serial.println(temp);
+  if (temp > 0 )
```

Comment:
Ok, but there's no way this will ever be less than zero. Technically it could be *equal* to zero, but the ADC cannot return a negative number, so this `if` is pretty meaningless.